### PR TITLE
Adjust phrasing of principles

### LINKED ISSUE #121: Align language in section titles
The language in the principle section titles is inconsistent. 

Many are positioned as statements of fact ("The Web is...", "The Web does not..."), but one is stated as a requirement ("The Web must...") and one is stated as a capability ("People can....").

Looking at the document with fresh eyes, I'd strongly recommend that they be normalised to be stated as requirements ("The Web must..."). Defining principles as statements of fact is confusing -- readers will misinterpret them as, well, statements of fact, and in some cases they are obviously NOT factual on the current Web.

### PR DESCRIPTION
**Pull request by mnot**
This is a minimal suggestion to reword the section titles to more clearly communicate that these are aspirations, not necessarily statements of fact.

Fixes #121.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -290,7 +290,7 @@ <h3 id="control" data-export="" data-dfn-type="dfn">
 
 <section>
   <h3 id="sustainable" data-export="" data-dfn-type="dfn">
-The web is an environmentally sustainable platform
+An ethical web is an environmentally sustainable platform
```
Comment (martinthomson) [inline index.html]:
```suggestion
An ethical web is environmentally sustainable
```

**Review by martinthomson — APPROVED** [review]
One possible tweak

**Review by hober — CHANGES_REQUESTED** [review]
I really dislike the use of the phrase "an ethical web" in this PR. It implies that there's more than one web, some of which are ethical and some of which are not. It makes the "There is one web" principle incoherent, and doesn't seem to enhance the clarity of the document elsewhere.

**Comment by mnot** [comment]
Oh, that's a shame - I really like this, because it's highlighting that there are multiple potential future Webs, and we can steer (to some degree) between them. I don't see the problem with "one web".

Possible alternatives:

1. Change them to a form like "The ethical web enables freedom of expression."
2. Change them to a form like "To be ethical, the web should enable freedom of expression."
3. Change them to a form like "The web should enable freedom of expression."

Thoughts?

**Comment by torgo** [comment]
We agreed to close this PR today as we don't agree this is the right wording.

**Pull request closed without merging**